server/src/instant/db/model/triple.clj

```diff
@@ -7,8 +7,10 @@
    [clojure.spec.alpha :as s]
    [instant.util.spec :as uspec]
    [instant.util.json :refer [->json]]
+   [instant.util.string :refer [multiline->single-line]]
    [instant.util.tracer :as tracer]
-   [instant.data.constants :refer [empty-app-id]])
+   [instant.data.constants :refer [empty-app-id]]
+   [instant.util.exception :as ex])
```

Comment:
This file is much easier to review with whitespace off https://github.com/instantdb/instant/pull/147/files?w=1